assets/capability/capabilities/alarm_generic.json

```diff
@@ -83,7 +83,7 @@
         "id": "alarm_generic_false",
         "title": {
           "en": "The generic alarm turned off",
-          "nl": "Het algemene alarm ging uit",
+          "nl": "Het algemene alarm gaat uit",
```

Comment:
Ik vind deze vertaling geen verbetering, de vorige was logischer want het alarm is al uit op dat moment